src/docs/product/performance/resources.md

```diff
@@ -138,6 +138,12 @@ If you want to investigate a specific page, click on it to open a sidebar showin
 
 <div style="position: relative; padding-bottom: calc(50.829268292682926% + 41px); height: 0;"><iframe src="https://demo.arcade.software/D96Pu2LMLHddyVCn3BOd?embed" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;color-scheme: light;"></iframe></div>
 
+### Image Samples
+
+When viewing an image resource, the resource summary page will also show the 5 largest images samples. If size information is not available, the samples will not be ordered in any particular order. If the images are publically accessible and the 'Enable Images' project settings is enabled, then the real images will be rendered in the UI.
```

Comment:
Should we specify what extensions we consider images?

Reply:
Yup, good idea!

Reply:
done!